bedrock/teach/templates/teach/smarton/surveillance.html

```diff
@@ -0,0 +1,344 @@
+{% block body_class %}smarton theme-surveillance{% endblock %}
+
+{% block smarton_content %}
+<article class="main">
+
+  {{ smarton_navbar(_('Government surveillance'), _('Get smart on government surveillance'), 'https://mzl.la/foo') }}
+
+  <section id="ask" class="section section-ask">
+    <div class="content">
+
+      <span class="smarton-hero">{% include 'teach/smarton/includes/blob-fpo.svg' %}</span>
+
+      <h2 class="section-title">{{ _('Ask yourself') }}</h2>
+      <p class="section-intro">
+        {# L10n: This is the first part of a question that is completed in three different ways. #}
+        {{ _('If you’re not under investigation, should the government be able to:') }}
+      </p>
+
+      {# L10n: These three fragments each complete the question, "If I'm not under investigation, should the government be able to ___?" #}
+      <ol class="big-list">
+        <li>
+          <p>{{ _('monitor your phone calls?') }}</p>
+          {% include 'teach/smarton/includes/list-number-1.svg' %}
+        </li>
+        <li>
+          <p>{{ _('read your emails and view the photos I send?') }}</p>
+          {% include 'teach/smarton/includes/list-number-2.svg' %}
+        </li>
+        <li>
+          <p>{{ _('see what you search for on the Web?') }}</p>
+          {% include 'teach/smarton/includes/list-number-3.svg' %}
+        </li>
+      </ol>
+
+    </div>
+  </section>
+
+  <section id="know" class="section section-know">
+    <div class="content">
+
+      <h2 class="section-title">{{ _('Government surveillance explained') }}</h2>
+
+      <p class="section-intro">
+        {{ _('Stakeouts have always been an essential tool for law enforcement and national security.') }}
+        {{ _('The difference now is that they’ve moved beyond binoculars, bugs and bad guys: surveillance has gone digital.') }}
+        {{ _('All that data we generate through our daily activities online allows governments to keep an eye on lots of people at once, in hopes of catching the ones who are up to no good.') }}
+        {{ _('It’s like grabbing a giant haystack just in case there might be a needle inside.') }}
+      </p>
+
+      <figure class="video">
+        <iframe width="620" height="349" src="https://www.youtube-nocookie.com/embed/RJpmbQTuItE?rel=0" frameborder="0" allowfullscreen>
+          <p><a href="https://www.youtube.com/watch?v=RJpmbQTuItE">{{ _('Watch this video') }}</a></p>
+        </iframe>
+      </figure>
+
+      <div class="with-sidebar">
+        <aside class="callout">
+          <blockquote>
+            <p class="quote">
+              {{ _('“Some amount of surveillance for law enforcement and intelligence, properly designed, is valuable to keep us safe. But when it’s too broad, it violates the privacy rights of millions of innocent people, and can even undermine our security.”') }}
+            </p>
+            <p class="attribute">— Chris Riley, {{ _('Head of Public Policy') }}, Mozilla</p>
```

Comment:
Please make the entire sentence localizable: they won't change the name, but they might need a different structure.
```
{{ _('— Chris Riley, Head of Public Policy, Mozilla') }}
```